setup.py

```diff
@@ -59,6 +59,7 @@
         "invenio_jsonschemas.schemas": ["inspire_records_schemas = inspire_schemas"],
         "invenio_search.mappings": ["records = inspirehep.search.mappings"],
         "invenio_celery.tasks": ["invenio_indexer = inspirehep.records.indexer.tasks"],
+        "invenio_db.alembic": ["inspirehep_new = inspirehep:alembic"],
```

Comment:
```suggestion
        "invenio_db.alembic": ["inspirehep = inspirehep:alembic"],
```

Reply:
It's called `inspirehep` already in inspire-next. They are separate environments but we should use different names to be safe.

Reply:
it doesn't matter they are different apps in different ``envs`` it's safe to have the same name :)

Reply:
As you wish, but I  do not like it.